Initial Wagtail Setup

simonprast opened the pull request:
**Checklist:**

 <!--- Make sure your PR is documented and tested before submission. Put an `x` in all the boxes that apply: -->
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?


**Type of Changes:**

 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [x] Breaking change (fix or feature that would cause existing functionality to change)


**What is the current behavior? (link to any open issues here)**
-  It is an empty github repository.


**What is the new behavior (if this is a feature change)?**
- The Wagtail page is now ready to be the base for our development.
#3 Wagtail Setup

simonprast committed:
Initial Wagtail Setup
Installation of Wagtail: http://docs.wagtail.io/en/v2.0/getting_started/
The Wagtail project was started with the command 'wagtail start charm'
For our project, I already changed the folder structure, removed the
default elasticsearch app and modified the model of the home app.

Aichnerc commented on charm/home/migrations/0001_initial.py:
```diff
@@ -0,0 +1,22 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```
Please explain what this class does

Aichnerc commented on charm/home/migrations/0002_create_homepage.py:
```diff
@@ -0,0 +1,58 @@
+from django.db import migrations
+
+
+def create_homepage(apps, schema_editor):
```
Please explain what this does.

Aichnerc commented on charm/home/migrations/0002_create_homepage.py:
```diff
@@ -0,0 +1,58 @@
+        hostname='localhost', root_page=homepage, is_default_site=True)
+
+
+def remove_homepage(apps, schema_editor):
```
Please explain what this does.

Aichnerc commented on charm/home/models.py:
```diff
@@ -0,0 +1,16 @@
+from wagtail.core.fields import RichTextField
+from wagtail.admin.edit_handlers import FieldPanel, TabbedInterface, ObjectList
+
+class CharmPage(Page):
```
Explain what this is.

Aichnerc reviewed (changes requested):
Please add some commentary, then it should be okay to merge. (Request new revision after your changes)

simonprast committed:
Add documentation to four files
As requested at the latest pull request, I added comments which explain
what the migration files in the home module are doing. Additionally, I
added comments in the Models file of the home app.

Aichnerc reviewed (approved):
Seems okay to me. But please verify your commits next time.

Aichnerc commented on charm/reports/models.py:
```diff
@@ -0,0 +1,52 @@
+
+class _S_ParagraphBlock(blocks.StructBlock):
+    statement = blocks.CharBlock(null=True, blank=True, help_text="Statement")
+    paragraph = blocks.RichTextBlock(null=True, blank=False, help_text="Manifest paragraph", features=['bold', 'italic', 'underline', 'strikethrough', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'blockquote', 'ol', 'ul', 'hr', 'embed', 'link', 'superscript', 'subscript', 'document-link', 'image', 'code'], classname="full")
```
The maximum length per line is 119 characters.

dermrsimon replied:
Affirmative.

dermrsimon commented:
ok i changed it

Aichnerc commented:
![image](https://user-images.githubusercontent.com/21159914/62994330-9b668780-be5b-11e9-96a7-77dda97fe3df.png)
